### PR TITLE
show views in new tab

### PR DESCRIPTION
**Pull request by chaudum**

### CODE REVIEW

File: app/scripts/controllers/views.js [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+              $scope.schema = views[0].schema;
+              $scope.name = views[0].name;
+              $state.go('views.view', {
+                schema: $scope.schema,
+                name: $scope.name
+              });
+            }
+
+            var idx,
+              name;
```
Comment (Autophagy) [inline app/scripts/controllers/views.js]: Any reason for the weird indentation here?

File: app/styles/views/_common.scss [hunk 2]
```diff
@@ -326,6 +326,13 @@ label:hover:before {
 	background-color: $cr-panel-danger-background !important;
 }
 
+.cr-panel--code {
+	border: $cr-panel-default-border !important;
+	border-radius: 3px;
+	background: $cr-panel-default-background !important;
+  font-family: $font-family-monospace;
```
Comment (Autophagy) [inline app/styles/views/_common.scss]: indentation

File: app/styles/views/_tablelist.scss [hunk 3]
```diff
@@ -1,9 +1,15 @@
-.table-side-bar{
+.table-side-bar {
 	display: flex;
 	flex-direction: column;
 	flex: 1 0 auto;
 }
 
+.table-side-bar-list {
+	flex: 1 1 auto;
+  overflow-x: hidden;
```
Comment (Autophagy) [inline app/styles/views/_tablelist.scss]: Indentation

File: app/scripts/controllers/views.js (hunk 1 above)
Comment (chaudum) [reply app/scripts/controllers/views.js]: that is the correct indentation (2 spaces) when having multiline variable declaration.

File: app/styles/views/_tablelist.scss (hunk 3 above)
Comment (chaudum) [reply app/styles/views/_tablelist.scss]: goddamn tabs

File: app/scripts/controllers/views.js (hunk 1 above)
Comment (Autophagy) [reply app/scripts/controllers/views.js]: Ah, cool, didnt know that.

**Pull request merged**